src/scheduler/resv_info.c

```diff
@@ -526,6 +526,7 @@ query_reservations(int pbs_sd, server_info *sinfo, struct batch_status *resvs)
 						 * from the parent reservation.
 						 */
 						resresv_ocr->resv->resv_state = RESV_CONFIRMED;
+						resresv_ocr->resv->is_running = 0;
```

Comment:
oops, this was missed in the other PR.  The other PR came from this PR.  I must have missed one line. I don't think it is a big deal.  I think it only affects when we are reconfirming a standing reservation when it is running.  It will just force all of the future occurrences back on the same nodes they were on.  It shouldn't be a problem.